src/Microsoft.Health.Fhir.Api/Features/Filters/ValidateParametersResourceAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Health.Fhir.Core.Exceptions;
+
+namespace Microsoft.Health.Fhir.Api.Features.Filters
+{
+    /// <summary>
+    /// Validate that the deserialized request body object is of type Hl7.Fhir.Model.Parameters.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
```

Comment:
Does this AttributeUsage need to be updated to [AttributeUsage(AttributeTargets.Class|AttributeTargets.Method)]?  I am not sure as the attribute seems to applied to a method in Reindex and working as is.

Reply:
Upon looking at the AttributeUsage documentation, it seems that filters with `[AttributeUsage(AttributeTargets.Class)]` should only work on the class level; I'm also confused how it's working with the POST method in MemberMatchController..

Anyhow, updated it to be `[AttributeUsage(AttributeTargets.Class|AttributeTargets.Method)]` as suggested.